zerver/tests/test_slack_importer.py

```diff
@@ -134,6 +134,7 @@ def test_users_to_zerver_userprofile(self, mock_get_data_file: mock.Mock) -> Non
                       "deleted": False,
                       "real_name": "John Doe",
                       "profile": {"image_32": "", "email": "jon@gmail.com", "avatar_hash": "hash",
+                                  "phone": "12345677868",
```

Comment:
Is this the format that Slack displays phone numbers in?  I.e. it's not the `+1-123-456-77-868` style?

Reply:
I have updated the tests for both `(605)234-1214` and `+1-123-456-77-868` style. It is majorly  `(605)234-1214` style, but any pattern would work here.